setup.py

```diff
@@ -96,7 +96,9 @@ def read_version_py(infname):
     #   an sftp extra in Tahoe-LAFS, there is no point in having one.
     # * Twisted 19.10 introduces Site.getContentFile which we use to get
     #   temporary upload files placed into a per-node temporary directory.
-    "Twisted[tls,conch] >= 19.10.0",
+    # * Twisted 22.8.0 added support for coroutine-returning functions in many
+    #   places (mainly via `maybeDeferred`)
+    "Twisted[tls,conch] >= 22.8.0",
```

Comment:
We could probably get a way with a much older one if we wanted -- we don't _need_ the `maybeDeferred` support, although I agree that's the best place to put it.

I'm just mentioning it because this likely has implications for other downstream projects?

Reply:
I reckon if you are upgrading Tahoe-LAFS you can probably upgrade Twisted too?

Reply:
Newer Twisted is also nice for making Eliot much more easy to use. This assumes Eliot is the logging library to use, of course—there's a good argument for OpenTelemetry.